gui/default/syncthing/core/syncthingController.js

```diff
@@ -780,6 +780,60 @@ angular.module('syncthing.core')
             return 'info';
         };
 
+        $scope.syncthingStatus = function () {
+            
+            var deviceCount = $scope.devices.length;
+            var syncCount = 0;
+            var notifyCount = 0;
+            var pauseCount = 0;
+            
+            //loop through all devices
+            for (var i = 0; i < $scope.devices.length; i++) {
+                var status = $scope.deviceStatus({
+                    deviceID:$scope.devices[i].deviceID
+                });
+                switch (status) {
+                    case 'syncing': syncCount++; break;
+                    case 'unknown': notifyCount++; break;
+                    case 'paused': pauseCount++; break;
+                    case 'unused': deviceCount--; break;
+                }
+            }
+
+            //loop through all folders
+            var folderListCache = $scope.folderList();
+            for (var i = 0; i < folderListCache.length; i++) {
+                var status = $scope.folderStatus(folderListCache[i]);
+                switch (status) {
+                    case 'syncing':
+                    case 'scanning':     //possible?
+                        syncCount++;
+                        break;
+                    case 'stopped':
+                    case 'unknown':
+                    case 'outofsync':
+                    case 'error':        //possible?
+                        notifyCount++;
+                        break;
+                }
+            }
+
+            //notifications
+            if ($scope.openNoAuth)                               notifyCount++;
+            if (!$scope.configInSync)                            notifyCount++;
+            if (Object.keys($scope.deviceRejections).length > 0) notifyCount++;
+            if (Object.keys($scope.folderRejections).length > 0) notifyCount++;
+            if ($scope.errorList().length > 0)                   notifyCount++;
+            if (!online)                                         notifyCount++;
+
+            // return order is important!
+            if (syncCount > 0) return 'sync';                     // at least one device or folder is syncing, or a folder is scanning
+            if (notifyCount > 0) return 'notify';                 // a device is unknown or a folder is stopped/unknown/outofsync/error or some other notification is open or gui offline
+            if (pauseCount === deviceCount-1) return 'pause';     // all used devices are paused except this one
+            return 'default';
```

Comment:
i feel its more important to show `sync` over `notify`. always.
if we return on first `notify` found (say device unknown) with a folder sync in progress - `notify` will show instead of `sync`.